docs_versioned_docs/version-ros2humble/ros/installation/remote_pc.mdx

```diff
@@ -64,5 +71,9 @@ ros2 run clearpath_generator_common generate_bash -s ~/clearpath
 Add the following line to your `~/.bashrc` file to automatically source the generated `setup.bash` file in new terminals:
 
 ```
-source /etc/clearpath/setup.bash
+source ~/clearpath/setup.bash
```

Comment:
Should this be the full path?
`/home/administrator/clearpath/setup.bash`

We have avoided `~/` in documentation, but I do not know why.

Reply:
@roni-kreinin thoughts? 

I am leaning towards leaving it as is because people will implement these instructions on their own PC's which may have different usernames (in the sim setup instructions). We could use $HOME if there is reason to do so.

Reply:
I think `~/` is fine, but using `$HOME` works too. The user may not always be `administrator`.